### PR TITLE
Removed Shared Mailboxes sentence in configure-email-forwarding.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/2348.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/configure-email-forwarding.md [hunk 1]
```diff
@@ -25,8 +25,6 @@ Email forwarding lets you to set up a mailbox to forward email messages sent to
 > [!IMPORTANT]
 > If you're using Microsoft 365 or Office 365 for business, you should configure email forwarding in the [Configure email forwarding](https://docs.microsoft.com/microsoft-365/admin/email/configure-email-forwarding)
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/configure-email-forwarding.md:26]:
```suggestion
> If you're using Microsoft 365 or Office 365 for business, you should configure email forwarding in the [Configure email forwarding](https://docs.microsoft.com/microsoft-365/admin/email/configure-email-forwarding).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor additional edit, please.

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**